src/StudentOrganizer.Infrastructure/Services/TeamService.cs

```diff
@@ -71,5 +75,49 @@ public async Task UpdateTeamName(UpdateTeamName command)
 
 			await _groupRepository.SaveChangesAsync();
 		}
-	}
-}
\ No newline at end of file
+
+        public async Task AddAssignment(AddAssignment command)
+        {
+            await _administratorService.ValidateAtLeastModerator(command.UserId, command.GroupId);
+
+            var assignment = await _assignmentRepository.GetAsync(command.AssignmentId);
```

Comment:
tworzymy assignment także nie szukamy go

Reply:
Ej to potrzebujemy AssignmentControllera? Bo tam są tworzone Assignmenty

Reply:
Można to zrobić w assignment controller. Btw ten kod który jest w assignment controller obecnie to można wywalić, bo jest niepoprawny